chapter_linear-networks/linear-regression-scratch_vn.md

```diff
@@ -275,7 +275,7 @@ def linreg(X, w, b):
 ## Defining the Loss Function
 -->
 
-## *dịch tiêu đề phía trên*
+## Xác định Hàm mất mát
```

Comment:
Trong đa số trường hợp thì `Define` dịch là `định nghĩa` sẽ sát hơn `xác định`. Mình bịa ra luôn chứ hông có liệt kê ra nhiều lựa chọn rồi chọn một cái.

cc @tiepvupsu

Reply:
Đúng rồi, cái này là `định nghĩa`  mà, `determine` mới là `xác định`.

Nhân tiện, bác @tanduong có nhiều comment đầy đủ quá, mời bác tham gia [slack của nhóm dịch](https://docs.google.com/forms/d/e/1FAIpQLScYforPRBn0oDhqSV_zTpzkxCAf0F7Cke13QS2tqXrJ8LxisQ/viewform) để tiện trao đổi thêm.

Reply:
Nếu nói trong toán thì từ `define` được dịch là `xác định` vẫn ổn. Trong mục này nói về cách kiếm hàm mất mát, tức là có nhiều hàm mất mát khác nhau và ta cần xác định trong bài toán này cần hàm mất mát như thế nào. Còn `định nghĩa` thường dùng để chỉ fact, lý thuyết. "Định nghĩa hàm mất mát" theo em cũng không phản ánh chuẩn nội dung trong mục.

Reply:
Chỗ này thật ra mình vẫn có chút bâng khuâng. Nhưng mình thấy ở mức độ của phần này của sách thì đang áp đặt hơn là thảo luận.

Reply:
> Nhưng mình thấy ở mức độ của phần này của sách thì đang áp đặt hơn là thảo luận.

Bạn giải thích rõ ý câu này được không? Ý bạn là tác giả đang áp đặt hay các reviewer đang áp đặt cách dịch? Mình có thể đọc lại cả mục rồi ra quyết định.

Reply:
Tham khảo thêm 
https://vi.wikipedia.org/wiki/L%E1%BA%ADp_tr%C3%ACnh_h%C6%B0%E1%BB%9Bng_%C4%91%E1%BB%91i_t%C6%B0%E1%BB%A3ng

các bạn sẽ thấy `định nghĩa lớp`, `định nghĩa đối tượng`.

Reply:
> > Nhưng mình thấy ở mức độ của phần này của sách thì đang áp đặt hơn là thảo luận.
> 
> Bạn giải thích rõ ý câu này được không? Ý bạn là tác giả đang áp đặt hay các reviewer đang áp đặt cách dịch? Mình có thể đọc lại cả mục rồi ra quyết định.

Ý mình là tác giả nêu ra định nghĩa chứ tác giả hông thảo luận nhiều các cách định nghĩa khác.